src/renderers/shaders/UniformsLib.js

```diff
@@ -66,6 +66,7 @@ THREE.UniformsLib = {
 
 	lights: {
 
+		"lightTexture" : { type: "t", value: new THREE.DataTexture( new Float32Array( 128 * 4 * 4 ), 4, 128, THREE.RGBAFormat, THREE.FloatType ) },
```

Comment:
I think the data texture should be nearest for filtering and no anisotropy.  Thus something like:
```
new THREE.DataTexture( new Float32Array( 128 * 4 * 4 ), 4, 128, THREE.RGBAFormat, THREE.FloatType, undefined, undefined, THREE.NearestFilter,  THREE.NearestFilter, 0 )
```

I wonder why DataTextures do not default to nearest, if they are data then interpolating doesn't make much sense.